app/views/layouts/application.html.erb

```diff
@@ -34,6 +34,16 @@
 
     <a href="#main-content" class="govuk-skip-link">Skip to main content</a>
 
+    <div id="global-cookie-message" class="govuk-cookie-banner" role="region" aria-label="cookie banner">
+      <div class="govuk-width-container">
+        <p class="govuk-cookie-banner__message">This service uses cookies to make the site simpler.</p>
+        <div class="govuk-cookie-banner__buttons">
+          <button class="govuk-button govuk-button--secondary" type="submit" id="accept-cookies">Accept cookies</button>
+          <%= link_to('Cookie information', cookies_path, class: "govuk-button govuk-button--secondary ") %>
+        </div>
+      </div>
+    </div>
```

Comment:
Should this default to hidden for the case of no-JS?

Reply:
I'll hide the accept button for non-JS users as per: https://github.com/alphagov/govuk_publishing_components/pull/948

Reply:
No JavaScript also means no cookies, but sure :).

Reply:
Not always, cookies can be set by response headers too